azure-pipelines/azure-pipeline.microbuild.before.yml

```diff
@@ -1,10 +1 @@
-steps:
-
-- task: MicroBuildSigningPlugin@3
-  inputs:
-    signType: $(SignType)
-    zipSources: false
-  displayName: Install MicroBuild Signing Plugin
-
-- task: MicroBuildSwixPlugin@4
-  displayName: Install MicroBuild Swix Plugin
```

Comment:
I didn't see where this went - I'm guessing parent template we are extending?

Reply:
It ended up here:

https://github.com/aspnet/LibraryManager/blob/77d22d5bcebe4b2773577a4076adc5ba01a2f4a3/azure-pipelines/official.yml#L38-L44